src/sentry/static/sentry/app/components/organizations/projectSelector.jsx

```diff
@@ -120,58 +116,14 @@ class ProjectSelector extends React.Component {
   }
 
   getProjects() {
-    const {organization, projects, multiProjects, nonMemberProjects} = this.props;
-
-    if (multiProjects) {
-      return [
-        sortArray(multiProjects, project => [!project.isBookmarked, project.name]),
-        nonMemberProjects || [],
-      ];
-    }
-
-    // Legacy
-    const {isSuperuser} = ConfigStore.get('user');
-    const unfilteredProjects = projects || organization.projects;
-
-    const filteredProjects = isSuperuser
-      ? unfilteredProjects
-      : unfilteredProjects.filter(project => project.isMember);
+    const {multiProjects, nonMemberProjects} = this.props;
 
     return [
-      sortArray(filteredProjects, project => [!project.isBookmarked, project.name]),
-      [],
+      sortArray(multiProjects, project => [!project.isBookmarked, project.name]),
+      nonMemberProjects || [],
     ];
   }
 
-  isControlled = () => typeof this.props.selectedProjects !== 'undefined';
```

Comment:
`<MultipleProjectSelector>` is the only component that depends on `<ProjectSelector>` and is always controlled, so remove any related code.